lib/foreman/renderer/scope/macros/helpers.rb

```diff
@@ -46,6 +46,14 @@ def to_json(data)
           def to_yaml(data)
             data.to_yaml
           end
+
+          apipie :method, "Generates formatted time from the Ruby Time object or Float object representing UNIX timestamp" do
+            required :time, Float, desc: 'Float object representing UNIX timestamp'
```

Comment:
Aren't unix timestamps usually integers? The line above it also suggests it accepts a `Time` object but the that isn't documented here.